src/components/structures/MatrixChat.tsx

```diff
@@ -1542,6 +1543,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 showNotificationsToast(false);
             }
 
+            cleanUpDraftsIfRequired();
```

Comment:
I think this point in the code is executed at the point we load the sync accumulator also, rather than just after the first real sync